### ISSUE TITLE
I am trying to use USB to VGA cable to connect two monitor with one PC

### ISSUE THREAD

**Issue by SATPAL-BHARDWAJ** [AdnanHodzic/displaylink-debian#720]
Only DisplayLink Service status is missing. All the prerequisites are matched, I have read the common issues, post installation guide as well. But I am confused. I am new to linux system. So, may be I have missed something. Please correct me. 

Ubuntu 16.04 LTS

Starting Debug ...

Did you read Post Installation Guide? http://bit.ly/2TbZleK [y/N] y

Did you read Troubleshooting most common issues? http://bit.ly/2Rofd0x [y/N] y

--------------- Linux system info ----------------

Distro: Ubuntu
Release: xenial
Kernel: 4.20.0-042000-lowlatency

---------------- DisplayLink info ----------------

Driver version: 1.11.0
DisplayLink service status: 
EVDI service version: 1.11.0

------------------ Graphics card -----------------

Vendor: i915
Subsystem: Core
VGA: Intel Corporation 2nd Generation Core Processor Family Integrated Graphics Controller (rev 09)
VGA (3D): 
X11 version: 1.18.4-0ubuntu0.12
X11 configs: /etc/X11/xorg.conf.d/20-displaylink.conf

-------------- DisplayLink xorg.conf -------------

File: /etc/X11/xorg.conf.d/20-displaylink.conf
Contents:
 Section "Device"
    Identifier  "DisplayLink"
    Driver      "modesetting"
    Option      "PageFlip" "false"
EndSection

-------------------- Monitors --------------------

Providers: number : 5
Provider 0: id: 0xcc cap: 0x9, Source Output, Sink Offload crtcs: 2 outputs: 3 associated providers: 4 name:modesetting
Provider 1: id: 0xa7 cap: 0x2, Sink Output crtcs: 1 outputs: 1 associated providers: 1 name:modesetting
Provider 2: id: 0x85 cap: 0x2, Sink Output crtcs: 1 outputs: 1 associated providers: 1 name:modesetting
Provider 3: id: 0x63 cap: 0x2, Sink Output crtcs: 1 outputs: 1 associated providers: 1 name:modesetting
Provider 4: id: 0x41 cap: 0x2, Sink Output crtcs: 1 outputs: 1 associated providers: 1 name:modesetting

**Comment by Barabazs** [comment]
How did you install displaylink?

**Comment by SATPAL-BHARDWAJ** [comment]
I just follow this https://www.reddit.com/r/Ubuntu/comments/ar025e/working_usbvga_adapters_for_1804/

I have upgraded my ubuntu from 16.04 to 18.04 LTS
Now, it shows everything fine, I mean display link status : up and running
But my second monitor still not detected.

I got to know that my USB to VGA cable should be displayLink device but currently I am using general cable. That might be the case.So, we can close this issue.

Thanks for your help.
